inc/Ui/PageDiff.php

```diff
@@ -66,7 +66,9 @@ public function compareWith($text = null)
             $changelog =& $this->changelog;
 
             // revision info object of older file (left side)
-            $this->RevInfo1 = new RevisionInfo($changelog->getCurrentRevisionInfo());
+            $info = $changelog->getCurrentRevisionInfo();
+            $info['media'] = false;
```

Comment:
I don't like this. If having media set is important, why doesn't getCurrentRevisionInfo() return it? Looking into that, why doesn't getRevisionInfo() return it? Finally why do these functions return an array which is then used to initialize a RevisionInfo object? Why do these functions not return the initialized objects?